recipes/stx/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
```

```suggestion
cmake_minimum_required(VERSION 3.8)
```

17 is only available since cmake 3.8

https://cmake.org/cmake/help/v3.8/prop_tgt/CXX_STANDARD.html#prop_tgt:CXX_STANDARD

---

Oh, good catch.